pkg/execute/defaultExecute.go

```diff
@@ -107,6 +112,13 @@ func WithTransformers(trans ...results.TransformerFunc) ExecuteOptions {
 	}
 }
 
+// WithEnvVars adds the provided env vars to the command
+func WithEnvVars(key, value string) ExecuteOptions {
```

Comment:
```suggestion
// WithEnvVar adds the provided env var to the command
func WithEnvVar(key, value string) ExecuteOptions {
```
After some reflection, wouldn't it be better to make the var name singular ? 
As we're adding a single env var, it would be better to name it `WithEnvVar`. What do you think about that ?

Reply:
@alexandrevilain good catch!
Since that function only adds one environment variable, there has more sense to use the singular on function name.
Thank!